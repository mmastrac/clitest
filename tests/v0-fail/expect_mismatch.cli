#!/usr/bin/env clitest --v0

$ printf "hello\n"
%EXPECT_FAILURE
! hello
