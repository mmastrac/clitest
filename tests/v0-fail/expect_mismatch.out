Running <root>/expect_mismatch.cli ...

printf "hello\n"
---
hello
---
❌ FAIL (output shouldn't match)
 → printf "hello\n"

<root>/expect_mismatch.cli FAILED
Error: expected failure, but passed

1/1 test(s) failed
